config/locales/en.yml

```
@@ -304,6 +304,24 @@ en:
   activemodel:
     errors:
       models:
+        publish/age_range_form:
+          attributes:
+            age_range_in_years:
+              blank: "You need to pick an age range"
+            course_age_range_in_years_other_from:
+              blank: "Enter an age in From"
+              not_a_number: "Enter a valid age in From"
```

```suggestion
not_a_number: &valid_message "Enter a valid age in From"
```

Then you could re-use the string via `*valid_message`

not sure if it's worth doing it here but just a little tip for re-usability

---

Ta